db/couchdb.js

```diff
@@ -52,19 +57,23 @@ const seedReviews = async () => {
            helpfulCount : faker.datatype.number(2000),
            abuseReported : faker.datatype.boolean()
```

Comment:
lol. I had that property set to false as a default in my FEC reviews: I thought someone in SDC could have fun implementing the functionality for clicking the report button on the client side.